include/matx/executors/device.h

```diff
@@ -84,41 +84,41 @@ namespace matx
           MATX_STATIC_ASSERT((sizeof(op) + sizeof(index_t) * Op::Rank()) <= CUDA_MAX_VAL_PARAM, 
               "Parameter buffer to device is limited to 4096B. Please break up your operator statement into multiple executions to limit the size of the parameters");
 
-          if constexpr (op.Rank() == 0) {
+          if constexpr (Op::Rank() == 0) {
```

Comment:
These are allowed by the standard, but MSVC also complains about it. I think it's a losing proposition to try to fix all of them since there are hundreds in the code.